frontend/src/app/signUp/IdentityVerification/PersonalDetailsForm.tsx

```diff
@@ -95,6 +95,12 @@ const PersonalDetailsForm = ({
     );
     showNotification(alert);
     setSaving(false);
+    let elementsWithErrors = Array.from(
+      document.querySelectorAll("[aria-invalid=true]")
+    );
+    (elementsWithErrors.find(
+      (element) => element.getAttribute("aria-hidden") !== "true"
+    ) as HTMLElement | null)?.focus();
```

Comment:
How does the focus work on an array? Does it go to the first element? It's a little easier to read in Organization form, not a big deal here though.

Reply:
So in `Array.find()` will return the first element that matches the condition. In this case, we want the first element that is not "hidden". 
This is more complicated then the other one because the date picker we use has a "hidden" field that takes the same properties that the other one does, so we need to find the first one.

Reply:
Gotcha! Filtering for "aria-hidden" !== "true" made sense, this is helpful to know that `.find` just returns the first element.

Reply:
I can imagine it being more readable to save the result of `.find` in a variable and then calling `.focus()` on it but that's just a matter of personal preference.